### PR TITLE
Initial pass of 1.6.8 release notes on master

### PR DESCRIPTION
**Pull request by George-Aeillo**
Initial pass of 1.6.8 release notes on master

### CODE REVIEW

File: content/en/docs/releasenotes/_index.md [hunk 1]
```diff
@@ -3,6 +3,21 @@ title: Release Notes
...
+- Refreshed component images for security.
+
+#### Component version updates:
+- Istio v1.19.0
+- Redis v7.0.12
+- Kiali v1.73.0
+- Rancher v2.7.6
+
+#### Fixes:
+- Fixed issue in Keycloak post installation.
```
Comment (rosemarymarano) [inline content/en/docs/releasenotes/_index.md:17]: -> Fixed a Keycloak post-installation issue.

File: content/en/docs/releasenotes/_index.md [hunk 2]
```diff
@@ -3,6 +3,21 @@ title: Release Notes
...
+
+#### Component version updates:
+- Istio v1.19.0
+- Redis v7.0.12
+- Kiali v1.73.0
+- Rancher v2.7.6
+
+#### Fixes:
+- Fixed issue in Keycloak post installation. 
+- Fixed issue with Cluster API URL overrides not working correctly.
```
Comment (rosemarymarano) [inline content/en/docs/releasenotes/_index.md:18]: -> Fixed an issue

File: content/en/docs/releasenotes/_index.md [hunk 3]
```diff
@@ -3,6 +3,21 @@ title: Release Notes
...
+- Fixed issue that caused the unecessary restart of OAM application pods.
```
Comment (rosemarymarano) [inline content/en/docs/releasenotes/_index.md:19]: -> Fixed an issue

File: content/en/docs/releasenotes/_index.md [hunk 4]
```diff
@@ -3,6 +3,21 @@ title: Release Notes
 weight: 2
 draft: false
 ---
+## v1.6.8
+#### Enhancements:
+- Refreshed component images for security.
+
+#### Component version updates:
+- Istio v1.19.0
+- Redis v7.0.12
```
Comment (rosemarymarano) [inline content/en/docs/releasenotes/_index.md:12]: Hmmm, I don't see Redis listed as a component here, https://verrazzano.io/latest/docs/setup/install/verify/softwares/ 
Can you find out more about it, for example, should it be added to this table? (Also, the other updates will need to be added as well.)

**Review by rosemarymarano — COMMENTED** [review]
Few edits/suggestions.

File: content/en/docs/releasenotes/_index.md (hunk 4 above)
Comment (George-Aeillo) [reply content/en/docs/releasenotes/_index.md:12]: Thanks! I will look into this and I will see if it should be added to the table. I will add the other updates to the table.

File: content/en/docs/releasenotes/_index.md (hunk 1 above)
Comment (George-Aeillo) [reply content/en/docs/releasenotes/_index.md:17]: Thanks!

File: content/en/docs/releasenotes/_index.md (hunk 3 above)
Comment (George-Aeillo) [reply content/en/docs/releasenotes/_index.md:19]: Thanks!

File: content/en/docs/releasenotes/_index.md (hunk 2 above)
Comment (George-Aeillo) [reply content/en/docs/releasenotes/_index.md:18]: Thanks!

File: content/en/docs/releasenotes/_index.md (hunk 4 above)
Comment (George-Aeillo) [reply content/en/docs/releasenotes/_index.md]: I was told to remove Redis from the release notes.

File: content/en/docs/releasenotes/_index.md (hunk 3 above)
Comment (rosemarymarano) [reply content/en/docs/releasenotes/_index.md:18]: One more to do.

**Pull request merged**